app/src/main/java/app/revanced/integrations/youtube/patches/components/ShortsFilter.java

```diff
@@ -130,62 +123,60 @@ public ShortsFilter() {
-                joinButton, subscribeButton, subscribeButtonPaused, suggestedActionPath,
+                joinButton, subscribeButton, subscribeButtonPaused, suggestedAction,
                 channelBar, fullVideoLinkLabel, videoTitle, reelSoundMetadata,
                 soundButton, infoPanel, actionBar
         );
 
         //
         // Action buttons
         //
-        var shortsLikeButton = new ByteArrayFilterGroup(
-                Settings.HIDE_SHORTS_LIKE_BUTTON,
-                "shorts_like_button"
-        );
-
-        var shortsDislikeButton = new ByteArrayFilterGroup(
-                Settings.HIDE_SHORTS_DISLIKE_BUTTON,
-                "shorts_dislike_button"
-        );
-
-        var shortsCommentButton = new ByteArrayFilterGroup(
-                Settings.HIDE_SHORTS_COMMENTS_BUTTON,
-                "reel_comment_button"
-        );
-
-        var shortsShareButton = new ByteArrayFilterGroup(
-                Settings.HIDE_SHORTS_SHARE_BUTTON,
-                "reel_share_button"
-        );
-
-        var shortsRemixButton = new ByteArrayFilterGroup(
-                Settings.HIDE_SHORTS_REMIX_BUTTON,
-                "reel_remix_button"
-        );
-
         videoActionButtonGroupList.addAll(
-                shortsLikeButton,
-                shortsDislikeButton,
-                shortsCommentButton,
-                shortsShareButton,
-                shortsRemixButton
+                // This also appears as the path item 'shorts_like_button.eml'
+                new ByteArrayFilterGroup(
+                        Settings.HIDE_SHORTS_LIKE_BUTTON,
+                        "reel_like_button",
+                        "reel_like_toggled_button"
+                ),
+                // This also appears as the path item 'shorts_dislike_button.eml'
+                new ByteArrayFilterGroup(
+                        Settings.HIDE_SHORTS_DISLIKE_BUTTON,
+                        "reel_dislike_button",
+                        "reel_dislike_toggled_button"
+                ),
+                new ByteArrayFilterGroup(
+                        Settings.HIDE_SHORTS_COMMENTS_BUTTON,
+                        "reel_comment_button"
+                ),
+                new ByteArrayFilterGroup(
+                        Settings.HIDE_SHORTS_SHARE_BUTTON,
+                        "reel_share_button"
+                ),
+                new ByteArrayFilterGroup(
+                        Settings.HIDE_SHORTS_REMIX_BUTTON,
+                        "reel_remix_button"
+                )
         );
 
         //
         // Suggested actions.
         //
-        suggestedActions.addAll(
+        suggestedActionsGroupList.addAll(
                 new ByteArrayFilterGroup(
                         Settings.HIDE_SHORTS_SHOP_BUTTON,
                         "yt_outline_bag_"
                 ),
+                new ByteArrayFilterGroup(
+                        Settings.HIDE_SHORTS_TAGGED_PRODUCTS,
+                        "PAproduct_listZ"
```

Comment:
Is there an image of the component's appearance or where it appears? If this is the same component as the other buttons, such as Shop (grey transparent button), maybe it's better and more straightforward for the user to either toggle the button on or off instead of having so many options for every possible kind the button can be.